kong/plugins/moesif/base64.lua

```diff
@@ -0,0 +1,136 @@
+
+local _M = {}
+
```

Comment:
@keyur9  are these changes copied from some place? usually a comment of the source would be helpful for future reference

Reply:
@dkm199 Sure will comment on the source details. For this, it is - https://github.com/iskolbin/lbase64